app/views/content/salaries-and-benefits.md

```diff
@@ -129,11 +129,13 @@ Their role is wide ranging, but includes leading and motivating teachers, and en
 
 You need qualified teacher status (QTS) to work in maintained primary, secondary and special schools in England. 
 
-If you do not have QTS, you can work in some schools as an unqualified teacher. 
+If you do not have QTS (for example, if you just have a PGCE), you can work in some schools as an unqualified teacher.
```

Comment:
Does this imply that all unqualified teachers have PGCEs @EllieNodder ? Does it need caveating with `... or do not have a qualification`? (This would need checking!)

Reply:
I think it's okay given we're saying 'for example', suggesting this is just one possible option? I also think that that could get complicated if we add something about 'or do not have a qualification' - are we suggesting then that you do not need any qualifications at all to work as an unqualified teacher?